docs/source/release_notes.rst

```diff
@@ -7,6 +7,7 @@ Release Notes
         * Changed the default parameter values for ``Elastic Net Classifier`` and ``Elastic Net Regressor`` :pr:`2269`
     * Fixes
     * Changes
+        * Updated ``start_iteration_callback`` to accept a pipeline instance instead of a pipeline class :pr:`2290`
```

Comment:
This is a bit of a nit-pick, but technically we should add a breaking change entry too, because this is going out in a different release from the pipeline API change.